src/azul/service/manifest_service.py

```diff
@@ -460,7 +460,7 @@ def _get_cached_manifest_file_name(self,
         else:
             seconds_until_expire = self._get_seconds_until_expire(response)
             if seconds_until_expire > config.manifest_expiration_margin:
-                _, _, default_file_name = object_key.rpartition('/')
+                default_file_name = object_key.rpartition('/')[2]
```

Comment:
I adopted the same [suggested convention](https://github.com/DataBiosphere/azul/pull/3183#discussion_r691445061), hopefully that's ok?

Reply:
Yes